agent/api/container/container.go

```diff
@@ -272,6 +274,19 @@ type MountPoint struct {
 	ReadOnly      bool   `json:"readOnly"`
 }
 
+// Log Router things
+type LogRouter struct {
+	Config               *LogRouterConfig `json:"config"`
+	EnableECSLogMetaData bool             `json:"enableECSLogMetaData"`
+	Type                 string           `json:"type"`
+}
+
+// Log Router things
```

Comment:
same as above. maybe something like "// LogRouterConfig specifies the config for the log router."